lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -0,0 +1,15 @@
+module GapIntelligence
+  class MerchantPricingTrend < Record
+    def initialize(attrs = {})
```

Comment:
@gtimti If we remove this overridden `initialize`, then according to existing logic in `record.rb`  `@attributes = raw.fetch('attributes', raw)` nothing will be returned as response does not have `attributes` node/key